packages/ubuntu_desktop_installer/lib/pages/keyboard_layout_page.dart

```diff
@@ -39,24 +47,22 @@ class _KeyboardLayoutPageState extends State<KeyboardLayoutPage> {
           return layout.code == info.layout;
         });
         if (_selectedLayoutIndex > -1) {
-          _selectedLayoutName =
-              keyboardModel.layouts[_selectedLayoutIndex].code!;
-          _selectedVariantIndex = keyboardModel
-              .layouts[_selectedLayoutIndex].variants!
+          _selectedVariantIndex = _selectedLayout!.variants!
```

Comment:
is the list of variants guaranteed to be non-null or should it be `.variants?.indexWhere(...) ?? -1`?

Reply:
You're right, of course. Fixed.